### PR TITLE
add: gopx-webui

### PR DESCRIPTION
**Pull request by GopalVerma1303**

### CODE REVIEW

**Comment by manuarora700** [comment]
@GopalVerma1303 can you please rebase with main?

**Comment by GopalVerma1303** [comment]
> @GopalVerma1303 can you please rebase with main?

Sure!

File: next.config.mjs [hunk 1]
```diff
@@ -6,6 +6,7 @@ const nextConfig = {
       { hostname: "sprintsocial.ai" },
       { hostname: "cdn.youprotab.com" },
       { hostname: "getaresu.me" },
+      { hostname: "webui.gopx.dev" },
     ],
```
Comment (ABSphreak) [inline next.config.mjs:10]: @GopalVerma1303 Add `tryhoverify.com` here too

**Review by ABSphreak — CHANGES_REQUESTED** [review]

File: next.config.mjs (hunk 1 above)
Comment (GopalVerma1303) [reply next.config.mjs]: Added

**Comment by GopalVerma1303** [comment]
No conflicts detected with the base branch. Please review and merge. Thank you.

**Pull request merged**